### PR TITLE
use csv extension and underscores instead of dashes

### PR DESCRIPTION
**Pull request by tank157**
[Pivotal Card](https://www.pivotaltracker.com/story/show/153451333)

### CODE REVIEW

**Comment by cap10morgan** [comment]
So they need the filenames to use a `.txt` extension (ugh, information loss) and underscores instead of dashes?

**Comment by cap10morgan** [comment]
And it wouldn't be feasible to change whatever else downstream wants `.txt` to also accept `.csv` as a much better fix?

**Comment by afsmythe** [comment]
Jumping in.. presently our VIP feed processing expects extension .txt and underscores. It's a minor inconvenience to have to change extension and underscore the resulting files post XML->CSV convertor. But, an inconvenience nonetheless. See conditions of acceptance in original card (https://www.pivotaltracker.com/story/show/151021203)

**Comment by tank157** [comment]
I could see the argument for making data-processor a bit more forgiving. But, considering how many places we have sending us data and how many other ways they already produce out-of-spec feed data, perhaps locking it down to one specific file name format in this case is wise, as it locks down the ways they can get something _almost but not quite_ right down too. In other words, give them less rope with which to, well, you know the rest of the saying.

**Comment by cap10morgan** [comment]
@afsmythe OK. Not that big of a deal in the grand scheme of things. But `.txt` tells one very little about the format of a file, whereas `.csv` says a whole lot more (and e.g. double-clicking such a file is much more likely to open it in a useful app). So on some timescale we should try to move in that direction. But it doesn't have to hold this up.

File: src/vip_feed_format_converter/csv.clj [hunk 1]
```diff
@@ -1,7 +1,14 @@
 (ns vip-feed-format-converter.csv
-  (:require [clojure.data.csv :as csv]
+  (:require [clojure.core.async :as async]
+            [clojure.data.csv :as csv]
             [clojure.java.io :as io]
-            [clojure.core.async :as async]))
+            [clojure.string :as str]))
+
+(defn ->filename
```
Comment (cap10morgan) [inline src/vip_feed_format_converter/csv.clj]: Not a blocker but I have a slight preference for dropping the `->` prefix here. I read a style guide thing once that said something along the lines of, "If you don't have something on both sides of that arrow, then just leave the arrow out of the name. A function name already implies that it returns something of that kind based on the args you give it," and I found that pretty compelling. Up to you either way.

**Pull request merged**